### PR TITLE
Create wai-aria/role/tree-roles.html

### PR DESCRIPTION
**Pull request by howard-e**
Closes https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/18

### CODE REVIEW

File: wai-aria/role/tree-roles.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    x
+    <ul role="group" data-testname="role is group (in treeitem)" data-expectedrole="group" class="ex">
+      <li role="treeitem" data-testname="role is treeitem (in group, in treeitem)" data-expectedrole="treeitem" class="ex">x</li>
+      <li role="treeitem">x</li>
+    </ul>
+  </li>
+  <li role="treeitem">x</li>
+</ul>
+
+<table role="treegrid" data-testname="role is treegrid" data-expectedrole="treegrid" class="ex">
```
Comment (cookiecrook) [inline wai-aria/role/tree-roles.html:27]: I think this one, or a new example, should include a collapsed row... Check if there is an APG example, but IIRC, that's aria-expanded=false on the row, with either hidden cell contents (e.g. display:none) or without any cell contents...

Comment (cookiecrook) [reply wai-aria/role/tree-roles.html:27]: The rest looks good. If you fine the collapsed row examples are difficult for some reason, you could also just include a to-do comment to get that one later. Thanks!

**Review by cookiecrook — APPROVED** [review]
Added suggestion, but approving either way.

File: wai-aria/role/tree-roles.html (hunk 1 above)
Comment (howard-e) [reply wai-aria/role/tree-roles.html:65]: Thanks for the suggestion! https://github.com/web-platform-tests/wpt/pull/41519/commits/ba5b36a52ffd4193928f7f65776629221f3063c8 now includes collapsible rows

**Comment by howard-e** [comment]
@cookiecrook re-requesting your review on the suggestion you shared

File: wai-aria/role/tree-roles.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    cursor: pointer;
+
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='12' height='12'%3E%3Cpolygon fill='black' points='2,0 2,10 10,5'%3E%3C/polygon%3E%3C/svg%3E%0A");
+    background-repeat: no-repeat;
+  }
+
+  [role="treegrid"] tr[aria-expanded="true"] > td:first-child::before,
+  [role="treegrid"] td[aria-expanded="true"]:first-child::before {
+    transform: rotate(90deg);
+  }
```
Comment (cookiecrook) [inline wai-aria/role/tree-roles.html:48]: Nice

File: wai-aria/role/tree-roles.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          }
+        } else {
+          // Only expand rows that are more "indented" than the ".expander" row. A higher aria-level indicates a
+          // higher "level"
+          if (nextSiblingLevel > expanderLevel) {
+            nextSibling.classList.add("hidden");
+          }
+        }
+
+        nextSibling = nextSibling.nextElementSibling;
```
Comment (cookiecrook) [inline wai-aria/role/tree-roles.html:142]: Cool. More functional that I expected. LGTM.

**Review by cookiecrook — APPROVED** [review]

**Pull request merged**